### PR TITLE
fix: allow registry acl lookups

### PR DESCRIPTION
**Pull request by axejintao**
# Summary

- allow development vault look ups from registry ACL

### CODE REVIEW

**Review by sajanrajdev — APPROVED** [review]
LGTM

File: src/registry/registry.service.ts [hunk 1]
```diff
@@ -94,6 +94,36 @@ export class RegistryService extends Service {
...
+    const developer = await this.registry.developer();
+    return this.getAuthorVaults(developer);
+  }
+
+  async getAuthorVaults(author: string): Promise<VaultRegistryEntry[]> {
+    if (!this.hasRegistry()) {
+      this.debug(
+        `${this.config.network} does not have a registry. No vaults are discoverable.`,
+      );
+      return [];
```
Comment (mrbasado) [inline src/registry/registry.service.ts:107]: not hooked up for now?

Comment (axejintao) [reply src/registry/registry.service.ts:107]: this is empty only when the registry does not exist, it is not possible to look up vaults if a registry is not deployed on a network

**Pull request merged**